### ISSUE TITLE
Feature request: specify a degree of rotation for portrait and landscape orientations

### ISSUE THREAD

**Issue by sudomain** [keymapperorg/KeyMapper#873]
**Developer TODO (don't remove)**
- [ ] create new branch. put issue number at start of name
- [ ] update documentation
- [ ] squash and merge
- [ ] delete branch

I guess not much of a feature request, but not quite a bug either: For the `Portrait mode` and `Landscape mode` it would be nice to able to specify:

1. a 0 or 180 degree rotation for portrait mode and
2. 90 or 270 degree rotation for landscape mode. 

This would be similar to how it can be done with the `Cycle through rotations` action to specify the degree of rotation instead of assuming 0 degrees for portrait and 90 degrees for landscape (atleast on the device I tested it on). Code could maybe be reused from `Cycle through rotations` action? Please consider

**Comment by sds100** [comment]
Yeah, this makes sense and would be simple to implement. I'm willing to add this feature.

**Comment by jambl3r** [comment]
@sds100 isn't this already implemented?

**Comment by sudomain** [comment]
It technically is not since the portrait/landscape actions do not have a way to specify degress. Looking back at this issue it seems like it's not really needed though. The solution is to just use `Cycle through rotations` and only select one of the checkboxes.

**Comment by jambl3r** [comment]
Ah okay. Yeah having a look now it's a constraint but not an action. If you're satisfied that it's not needed I can close this
